src/main/webapp/app/admin/health/health.component.html

```diff
@@ -16,20 +17,18 @@ <h2>
             <tbody *ngIf="health">
                 <tr *ngFor="let componentHealth of health.components | keyvalue">
                     <td>
-                        <span>
-                            {{ 'health.indicator.' + componentHealth.key | artemisTranslate }}
-                            <a *ngIf="componentHealth.value?.details?.url" class="badge bg-info" [href]="componentHealth.value?.details.url">
-                                {{ componentHealth.value?.details.url }}
-                            </a>
-                        </span>
+                        {{ 'health.indicator.' + componentHealth.key | artemisTranslate }}
+                        <a *ngIf="componentHealth.value?.details?.url" class="badge bg-info" [href]="componentHealth.value!.details!.url">
+                            {{ componentHealth.value!.details!.url }}
+                        </a>
                     </td>
                     <td class="text-center">
-                        <span class="badge" [ngClass]="getBadgeClass(componentHealth.value?.status)" jhiTranslate="{{ 'health.status.' + componentHealth.value?.status }}">
-                            {{ componentHealth.value?.status }}
+                        <span class="badge" [ngClass]="getBadgeClass(componentHealth.value!.status)" jhiTranslate="{{ 'health.status.' + componentHealth.value!.status }}">
+                            {{ componentHealth.value!.status }}
                         </span>
                     </td>
                     <td class="text-center">
-                        <a class="hand" (click)="showHealth(componentHealth)" *ngIf="componentHealth.value?.details">
+                        <a class="hand" (click)="showHealth({ key: componentHealth.key, value: componentHealth.value! })" *ngIf="componentHealth.value!.details">
```

Comment:
Why is this object recreated? Shouldn't the two versions be equal in that detail?

Reply:
Not necessary since generated from jhipster